app/src/main/res/layout/login_fragment.xml

```diff
@@ -39,60 +39,67 @@
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
                         android:hint="@string/email"
-                        android:text="@={ login.email }"
-                        android:inputType="textEmailAddress"
                         android:imeOptions="actionNext"
-                        app:validateType='@{"email"}' />
+                        android:inputType="textEmailAddress"
+
+                        app:validateType='@{"email"}'
+
+                        android:text="@={ login.email }"
```

Comment:
Please remove the unnecesaary new lines and spacing.

Reply:
This was the change I was talking about not being still addressed